compiler/sem.nim

```diff
@@ -621,7 +621,8 @@ proc testExamples(c: PContext) =
   if os.execShellCmd(os.getAppFilename() & " " & backend & " --nimcache:" & nimcache & " -r " & outp) != 0:
     quit "[Examples] failed: see " & outp
   else:
-    removeFile(outp)
+    # keep generated source file `outp` to allow inspection.
+    echo "success for genarated " & outp
```

Comment:
The compiler doesn't use ``echo`` and you have a typo in that message too. Come on.

Reply:
done (btw, echo is used in a few places in compiler; i thought about using msgWriteln but hintSuccess was a good fit; and was actually unused up until now)